### DISCUSSION TITLE
[Feature Request] Automatic audio timing setup/GlobalOffsetSeconds with wired Mic

### DISCUSSION THREAD

**Discussion by LightningSkunk** [TeamRizu/OutFox#257] [03 - Feature Requests]
What would be really sweet is if Outfox could take microphone input from a wired mic in order to automatically set the machine GlobalOffsetSeconds. Not unlike how Onkyo Audyessy receivers do when configuring speaker volume and delay. This could then automatically "zero-out" the offset timing based on where your dance pad is positioned from your speakers.
if combined with at least the ability to manually adjust for millisecond visual delay based on a reference like https://displaylag.com/display-database/  or one of these things: http://leobodnar.com/shop/index.php?main_page=product_info&products_id=212 , one could dial in any setup perfectly every time.

An example of manually getting this value with Audacity
https://manual.audacityteam.org/man/latency_test.html

Maybe you could even call it something cheeky like Outfox Autotune :P

**Comment by Scraticus** [comment]
You are talking about distance calculation of audio to the centre of the dance pad?

I believe this is also done with my Yamaha Receiver as well. Will look into it but it will be low priority.
